rustfmt-core/rustfmt-lib/tests/source/mods-relative-paths/mod_b.rs

```diff
@@ -0,0 +1,7 @@
+fn
```

Comment:
Should the `mod_b` file be added to the [skip list](https://github.com/rust-lang/rustfmt/blob/a9d7cd572cc07dccab703c8ce5df4ecbd2f085d4/rustfmt-core/rustfmt-lib/src/test/mod.rs#L23), along with a `recursive` added to the `lib.rs` files?

Reply:
Right, good catch. Thank you!

Reply:
Fixed in https://github.com/rust-lang/rustfmt/pull/4131/commits/b873aea85d5b66bfb82a10879d73767349333ccd.